README.md

```diff
@@ -10,7 +10,16 @@ Crystal is a programming language with the following goals:
 * Have compile-time evaluation and generation of code, to avoid boilerplate code.
 * Compile to efficient native code.
 
-## Build
+## Contributing to These Docs
```

Comment:
Probably it's not clear, but I think it'd be a good thing to change. IMO both the API reference and the book are documentation, so calling the latter "docs" is naturally confusing. 

What's your opinion about changing all "docs" references throughout the website and inside this repo to "book", "Crystal Book", or similar?